openmc/data/multipole.py

```diff
@@ -0,0 +1,628 @@
+from six import string_types
+
+from numbers import Integral, Real
+
+import h5py
+import numpy as np
+from scipy.special import wofz
+from six import string_types
+
+from . import WMP_VERSION
+from .data import K_BOLTZMANN
+import openmc.checkvalue as cv
+from openmc.mixin import EqualityMixin
+
+
+# Formalisms
+FORM_MLBW = 2
+FORM_RM   = 3
+FORM_RML  = 7
+
+# Constants that determine which value to access
+MP_EA = 0       # Pole
+
+# Reich-Moore indices
+RM_RT = 1       # Residue total
+RM_RA = 2       # Residue absorption
+RM_RF = 3       # Residue fission
+
+# Multi-level Breit Wigner indices
+MLBW_RT = 1     # Residue total
+MLBW_RX = 2     # Residue compettitive
+MLBW_RA = 3     # Residue absorption
+MLBW_RF = 4     # Residue fission
+
+# Polynomial fit indices
+FIT_T = 0       # Total
+FIT_A = 1       # Absorption
+FIT_F = 2       # Fission
+
+
+def _faddeeva(z):
+    """Evaluate the complex Faddeeva function.
+
+    Technically, the value we want is given by the equation:
+    w(z) = I/Pi * Integrate[Exp[-t^2]/(z-t), {t, -Infinity, Infinity}]
```

Comment:
Please use [`:math:` role for Sphinx](http://www.sphinx-doc.org/en/1.4.8/ext/math.html#role-math)